controllers/nginx/rootfs/etc/nginx/lua/error_page.lua

```diff
@@ -8,26 +8,27 @@ local get_upstreams = upstream.get_upstreams
 local random = math.random
 local us = get_upstreams()
 
-function openURL(status)
+function openURL(headers, status)
     local httpc = http.new()
 
+    headers["X-Code"] = status or "404"
+    headers["X-Format"] = headers["Accept"] or "html",
+
     local random_backend = get_destination()
     local res, err = httpc:request_uri(random_backend, {
         path = "/",
         method = "GET",
-        headers = {
-            ["X-Code"] = status or "404",
-            ["X-Format"] = ngx.var.httpAccept or "html",
```

Comment:
Nothing. The idea of this PR is to pass all the headers from the original request to the custom error backend.